### PR TITLE
Change on Boarding Doc

### PR DESCRIPTION
**Pull request by LianwMS**

### CODE REVIEW

File: doc/00-onboarding-guide.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
-# Onboarding process for using Azure Cli Codegen (autorest.az)
+# Onboarding Process for Using Azure Cli Code-gen (autorest.az)
```
Comment (RodgeFu) [inline doc/00-onboarding-guide.md:1]: there is another file called "onboarding-guide.md" at the same folder. please clear that folder and link to this one.

File: doc/00-onboarding-guide.md [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+Thanks for selecting Azure Cli Code-gen as approach to onboarding Azure Cli for your RP. No matter you want add a new feature to existing PR or onboard a new RP. This document will provide the guideline for your on-boarding process.
 
-Send mail to amecodegen@microsoft.com when your swagger is ready, then we will
+## Step 1. Send request to AME team in Github
+AME team use github issue as a centralized entry point to trigger the process for new RP onboarding or add feature in existing RP. Please go to [Azure-Cli](https://github.com/Azure/azure-cli) to create a new github issue and following the onboarding template first. 
+With the selection result in template, if you select Azure cli Code-gen as the approach, you will be re-direct back to this document later.
+> ** we are improving the process here with Azure CLI team soon 
 
+## Step 2. Quick Validation
+If it is your first time to use Azure Cli code-gen, please discuss with the contact person. He/She will:
```
Comment (RodgeFu) [inline doc/00-onboarding-guide.md:11]: >the contact person [](start = 72, length = 18)

please mention a contact person will be assigned to help in Step1.

File: doc/00-onboarding-guide.md [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
 
-## Step 4. Onboarding to azure-cli
-1. Follow [azure-cli guidance](https://github.com/Azure/azure-cli/blob/dev/doc/onboarding_guide.md) to onboard to azure-cli
-> ** we are improving the process here with Azure CLI team now
-
-## Step Next. CLI is ready when swagger is ready
-Autorest.az has been integrated into swagger PR. It means you can start reviewing your CLI code/command immediately when your swagger PR is filed for your new features/changes, so your new features/changes can have Azure CLI support very soon. 
-
-Hopefully, **your CLI is ready to merge/release when your swagger is ready to merge. :shipit: :+1:** 
+## Step 5. Run Linter and Style
```
Comment (RodgeFu) [inline doc/00-onboarding-guide.md:23]: >Run Linter and Style [](start = 11, length = 20)

let's merge this into Step4 to make things look simpler

File: doc/00-onboarding-guide.md [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
+1. Run linter and style check for the code generated and make sure all of them can pass. (Refer to [[here](how-to-generate.md#through-our-pre-prepared-docker) for how to run linter and style check])
 
-Check this [10-minutes video](https://msit.microsoftstream.com/video/71cea3ff-0400-a9f4-01b4-f1ea9e9b130e) for more details
-> ** we are working on the process here with Azure CLI and swagger team now
+## Step 6. Code review with Cli team
+1. Discusses with your contact person to ask for code review of the Az Cli code.
 
+## Step 7. Onboarding to azure-cli
```
Comment (RodgeFu) [inline doc/00-onboarding-guide.md:29]: >Onboarding to azure-cli [](start = 11, length = 23)

please merge this with Step6

File: doc/how-to-generate.md [hunk 5]
```diff
@@ -42,6 +66,18 @@ We have pre-prepared docker image for you to use autorest.az easily. (Please mak
...
+    * run azdev linter
+    ``` bash
+    > azdev linter {service_name}
+    ```
+
+6. Run style check on generated code
+    * run azdev style
+    ``` bash
+    > azdev style {service_name}
+    ```
```
Comment (RodgeFu) [inline doc/how-to-generate.md:79]: please merge this with test to make things look simpler.thx

File: doc/00-onboarding-guide.md (hunk 1 above)
Comment (LianwMS) [reply doc/00-onboarding-guide.md:1]: remove the other one

---
In reply to: [449503255](https://github.com/Azure/autorest.az/pull/461#discussion_r449503255) [](ancestors = 449503255)

File: doc/00-onboarding-guide.md (hunk 2 above)
Comment (LianwMS) [reply doc/00-onboarding-guide.md:11]: Added

---
In reply to: [449503755](https://github.com/Azure/autorest.az/pull/461#discussion_r449503755) [](ancestors = 449503755)

File: doc/00-onboarding-guide.md (hunk 3 above)
Comment (LianwMS) [reply doc/00-onboarding-guide.md:23]: ok

---
In reply to: [449504555](https://github.com/Azure/autorest.az/pull/461#discussion_r449504555) [](ancestors = 449504555)

File: doc/how-to-generate.md (hunk 5 above)
Comment (LianwMS) [reply doc/how-to-generate.md:79]: resolve

---
In reply to: [449505426](https://github.com/Azure/autorest.az/pull/461#discussion_r449505426) [](ancestors = 449505426)

File: doc/00-onboarding-guide.md (hunk 4 above)
Comment (LianwMS) [reply doc/00-onboarding-guide.md:29]: resolved

---
In reply to: [449504853](https://github.com/Azure/autorest.az/pull/461#discussion_r449504853) [](ancestors = 449504853)

**Pull request merged**